website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Amazon Web Services (AWS)"
+linkTitle: "AWS"
+weight: 10
+---
+
+## Control Provisioning with Labels
+
+The [Provisioner CRD]({{< ref "provisioner-crd.md" >}}) supports defining
+instance types, node labels, and node taints.
+
+For certain well-known labels (documented below), Karpenter will provision
+nodes accordingly. For example, in response to a label of
+`topology.kubernetes.io/zone=us-east-1c`, Karpenter will provision nodes in
+that availability zone.
+
+Regarding AWS, 3 types of provisioning constraints are recognized: 

```

Comment:
"Regarding AWS" seems like an awkward way to start a sentence to me. Also -- specifying the number in the sentence will easily become stale as we add more things to this list.

Reply:
this will be resolved after we refactor the categories